workspace/test-utils/package.json

```diff
@@ -26,11 +26,12 @@
   "dependencies": {
     "@starbeam/resource": "workspace:^",
     "@starbeam/runtime": "workspace:^",
+    "@starbeam/shared": "workspace:^",
     "@starbeam/universal": "workspace:^",
     "@starbeam/verify": "workspace:^"
   },
   "devDependencies": {
     "@starbeam-dev/build-support": "workspace:*",
-    "rollup": "^3.20.6"
+    "rollup": "^3.26.3"
   }
 }
```

Comment:
I think it'd be cool to publish test-utils, so that folks testing starbeam can also use these abstractions.

_however_, we'd need a way to pass in different expect functions, because not everyone uses vitest.

_DEF_ not for this PR though. lol